### PR TITLE
Feat/readme

### PR DESCRIPTION
**Pull request by efelem**
Updating readme. 
+ Ability to set job price (important for the chatBot demo)

### CODE REVIEW

**Comment by efelem** [comment]
There is also the async lock added here to prevent nonce collision.

**Comment by efelem** [comment]
There is also a change in the build to delete the parcel cache folder (otherwise the model is not updated correctly when using pnpm link to dev)

**Comment by Lymnah** [comment]
@efelem Missing README update no?

**Review by valentinpollart — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -123,6 +144,92 @@ In order to cancel a job you can use :
...
+At the end of the script, we call main() to start the asynchronous execution of the script.
+
+
+
+
+Florin Dzeladini
+Will this wait for end of execution of main ? Or do I need to add a setTimeOut ? 
+Yes, the modified code I provided with the async function main() and main() function call at the end will wait for the end of the execution of main() before exiting the script.
+
+When you call main() at the end of the script, the async keyword indicates that it will return a Promise. However, since there is no code that explicitly awaits this Promise, Node.js will wait for the Promise to resolve or reject before exiting the script. Therefore, there is no need to add a setTimeout function.
```
Comment (valentinpollart) [inline README.md:222]: Not sure this has been left here voluntarily

**Review by valentinpollart — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (efelem) [reply README.md:222]: I guess not ^^

**Review by valentinpollart — APPROVED** [review]

**Pull request merged**